src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -530,5 +530,20 @@ private void displayConfidenceLevel()
                 this.lblConfidenceLevel.Margin = new Thickness(left, -15, 0, 0);
             }
         }
+
+        private void zoomScaleLevel_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        {
+            Slider slider = (Slider)sender;
+
+            //Since the percentage goes from 10 to 300, the value is decremented by 10 to standardize. 
+            double percentage = slider.Value - 10;
```

Comment:
Now that it starts with 10%, thats why we minus 10 as a starting point?

Reply:
Yes, to facilitate the calculation.